Modules/Filtering/ImageFeature/test/itkLaplacianImageFilterTest.cxx

```diff
@@ -59,7 +59,7 @@ itkLaplacianImageFilterTest(int argc, char * argv[])
   ITK_TRY_EXPECT_NO_EXCEPTION(reader->Update());
 
   // Assign a spacing other than [1,1] for testing
-  itk::Vector<float, 2> spacing{ { 0.5, 5.0 } };
+  itk::Vector<float, 2> spacing(0.5);
```

Comment:
This is changing the test parameters. Spacing should be non-uniform [0.5, 5.0].

Reply:
What about `itk::MakeVector(0.5f, 5.0f)`?  😃

Reply:
:100: Good catch. Addressed in 3265872.

Reply:
> What about itk::MakeVector(0.5f, 5.0f)? smiley

@N-Dekker I personally do not have a preference other than minimizing the burden on the contributors. I barely code in C++, and my knowledge on modern C++ practice or efficiency aspects is limited. I would suggest appropriately documenting these in the class headers, or even better, the ITK SW Guide, so that we can always be aware of the best/most readable/efficient practices, taking possibly into account that compilers need to be happy, and stick to them when contributing, and let the other ways be tested in the test files.

So for now, I am leaving as it is unless Sean says the current solution does not satisfy the compiler.